### ISSUE TITLE
[BUG] exo fails parsing "security_opt" from docker compose file

### ISSUE THREAD

**Issue by catleeball** [deref/exo#279]
**Describe the bug**

When running `exo apply` or `exo run` on a `docker-compose.yml` file containing the `security_opt` option.

Here's the compose file I used: https://gist.github.com/catleeball/b18c07c17c674d26f7e443cbf01b2089

And here's the error exo printed: https://gist.github.com/catleeball/8a72f066c0ea415e6ebd4529aa8843fd
 
**To Reproduce**
1. Make compose file that includes `security_opt` options
2. Run `exo apply` or `exo run`
3. Exo throws a parser error

**Expected behavior**

Exo starts the containers according to the compose file.

**System Info (please complete the following information):**
 - OS: linux ( Ubuntu 21.04 with kernel 5.11 )
 - Component: cli
 - Version: 2021.08.31

**Additional context**

Here's the exo error message inline in case its helpful to surface this bug in future searches:

```
Error: applying manifest: parsing: [36:5] unknown field "security_opt"
      34 |     image: photoprism/photoprism:latest
      35 |     restart: unless-stopped
    > 36 |     security_opt:
               ^
      37 |       - seccomp:unconfined
      38 |       - apparmor:unconfined
      39 |     ports:
applying manifest: parsing: [36:5] unknown field "security_opt"
      34 |     image: photoprism/photoprism:latest
      35 |     restart: unless-stopped
    > 36 |     security_opt:
               ^
      37 |       - seccomp:unconfined
      38 |       - apparmor:unconfined
      39 |     ports:
```

**Comment by kendru** [comment]
@catleeball Thank you so much for this bug report! We are working hard to make sure that we get as close as possible to supporting all of the features that docker-compose supports, but `security_opt` is one that we have not implemented yet. I'll make sure that we get support added as quickly as possible! I really appreciate you linking to the full Docker Compose file so that we can make sure that we support everything that you are using in there.

I will let you know when we cut a release that includes this fix.

**Comment by catleeball** [comment]
Thanks for the update, and for making this cool bit of software! :)

**Comment by brandonbloom** [comment]
This is now available in the 2021.09.07 release.
